### DISCUSSION TITLE
i18n feature?

### DISCUSSION THREAD

**Discussion by JeremyWu917** [lepoco/wpfui#392] [Ideas]
Is there any plan to support i18n in wpfui?

**Comment by pomianowski** [comment]
How? By which method/technology? It would be nice to provide more information...

WPF supports globalization by default
https://learn.microsoft.com/en-us/dotnet/desktop/wpf/advanced/wpf-globalization-and-localization-overview?view=netframeworkdesktop-4.8&viewFallbackFrom=netdesktop-6.0
